_data/meltano/utilities/superset/apache.yml

```diff
@@ -0,0 +1,32 @@
+name: superset
+namespace: superset
+label: Apache Superset
+description: A modern, enterprise-ready business intelligence web application.
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/apache/superset
+variant: apache
+pip_url: apache-superset==1.5.0 markupsafe==2.0.1
+requires:
+  files:
+  - name: files-superset
+    variant: meltano
+settings:
+- name: home
+  value: $MELTANO_PROJECT_ROOT/.meltano/run/superset
+  description: The home directory where Superset will store data.
+- name: additional_dependencies
```

Comment:
I'd rather remove this and instruct people to add dependencies to `pip_url`, like we've done with Airflow: https://docs.meltano.com/guide/production#airflow-orchestrator:

> For this to work, the [psycopg2 package](https://pypi.org/project/psycopg2/) will also need to be installed alongside [apache-airflow](https://pypi.org/project/apache-airflow/), which you can realize by adding psycopg2 to airflow’s pip_url in your [meltano.yml project file](https://docs.meltano.com/concepts/project#meltano-yml-project-file) (e.g. pip_url: psycopg2 apache-airflow) and running [meltano install orchestrator airflow](https://docs.meltano.com/reference/command-line-interface#install).

Otherwise these dependencies will be reinstalled every time Superset is initialized.